tests/e2e/online_tests.py

```diff
@@ -4,40 +4,56 @@
 import time
 import unittest
 
-from credentials import waremart_password, waremart_user
-from tests.e2e import setup_e2e
+from typing import Optional
 
-debug_log = "--logging-level=DEBUG"
-indexing_sleep_time = 1  # wait 1 second to confirm server has indexed updates
+try:
+    from tests.e2e.credentials import database_password, database_user
+except ModuleNotFoundError:
+    database_user = None
+    database_password = None
+    print("Could not import datasource creds from credentials file")
+
+from tests.e2e import setup_e2e
 
-project_name = "not-default-name"
-group_name = "test-ing-group"
-workbook_name = "namebasic"
 
 # to run this suite
 # pytest -q tests/e2e/online_tests.py
 # you can either run setup with a stored credentials file, or simply log in
 # before running the suite so a session is active
 
-# alpodev
-parent_location = "WAM"
-project_name = "Developer Platform"
+debug_log = "--logging-level=DEBUG"
+indexing_sleep_time = 1  # wait 1 second to confirm server has indexed updates
+
+# object names
+unique = str(time.gmtime().tm_sec)
+
+default_project_name = "default-proj" + unique
+group_name = "test-ing-group" + unique
+workbook_name = "wb_1_" + unique
+
+parent_location = "parent" + unique
+project_name = "test-proj-" + unique
 
 server_admin = False
 site_admin = True
 project_admin = True
+extracts_enabled = False
+use_tabcmd_classic = False # toggle between testing using tabcmd 2 or tabcmd classic
 
 
 def _test_command(test_args: list[str]):
     # this will raise an exception if it gets a non-zero return code
     # that should bubble up and fail the test?
-    # dist/exe calling_args = [setup_e2e.exe] + test_args + [debug_log]
+    # using tabcmd 2
     calling_args = ["python", "-m", "tabcmd"] + test_args + [debug_log] + ["--no-certcheck"]
+
+    # call the executable directly: lets us drop in classic tabcmd
+    if use_tabcmd_classic:
+        calling_args = ["C:\\Program Files\\Tableau\\Tableau Server\\2023.3\\extras\\Command Line Utility\\tabcmd.exe"] + test_args + ["--no-certcheck"]
     print(calling_args)
```

Comment:
This alert is complaining specifically because it could print the embedded database password when called in test_wb_publish_embedded(self)

Since I would like to be able to use this test suite in github, perhaps we can add a check to skip printing if test_args contains "--db-password"?